packages/components/src/loader/src/Loader.css

```diff
@@ -2,8 +2,8 @@
     --animation-duration: 2s;
     opacity: 0;
     transition: opacity var(--o-ui-easing-duration-2) var(--o-ui-easing-productive);
-    width: var(--o-ui-sz-5);
-    height: var(--o-ui-sz-5);
+    width: var(--hop-space-320);
+    height: var(--hop-space-320);
```

Comment:
For width and heights, we just hard code the size since we don't have the semantic tokens yet.